index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <nav>
+        <img src="/Images/Light.png"/>
+    </nav>
+    <header class="bar">
+        <img src="/Images/app_bar.png"/>
+    </header>
+    <div class="container">
+        <section class="head">
+            <h2> 
```

Comment:
- [ ] **Kindly as suggested by the previous reviewer, 👆 👆 ** the <header> element represents the topmost section of the page, and the <section> elements are used to divide the content into meaningful sections. By utilizing these elements properly. 👍 

   - I think it would be better for the container of this section to use a `header` tag. 👌😎
    Typically the header tag is used as a page header, it contains the site's **title** or **logo** (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info. 👍 

example:
```
        <header class="navbar-left">
            <a class="logo-text">My Logo</a>
            <nav>
                <img src="./icons/Icon - Menu.png" alt="burguer menu">
             </nav>
        </header>

Reply:
sorry your reply seems to be cut at the start. the container class I have holds the text too so... Should I make the container  class be for the header and pull all the text inside too, or should I make a new container in header that includes the logo, and turn it into header? 

Something like
```
    <header class="bar-container">
         <nav>
            <img src="/Images/Light.png"/>
         </nav>
        <img src="/Images/app_bar.png"/>
    </header>
    <div class="container">
        <section class="head">
            <h2> 
                Hey there. I’m Santiago. 
                <div class="title"> I’m a software developer </div>
            </h2>
```

or something like

```
    <nav>
        <img src="/Images/Light.png"/>
    </nav>
    <header class="container">
        <img src="/Images/app_bar.png"/>
        <section class="head">
            <h2> 
                Hey there. I’m Santiago. 
                <div class="title"> I’m a software developer </div>
            </h2>
```